src/Symfony/Bundle/FrameworkBundle/DependencyInjection/Configuration.php

```diff
@@ -84,6 +84,14 @@ public function getConfigTreeBuilder()
                                 'base_urls' => array_values(array_unique(array_merge($config['base_urls']['http'], $config['base_urls']['ssl']))),
                             );
                         }
+                    } else {
```

Comment:
this is the wrong place to set it (you are enabling it only when ``templating`` is set, while it should always be set)